docs/detections/alerts-view-details.asciidoc

```diff
@@ -123,33 +210,86 @@ When gathering threat intelligence data for an alert, {elastic-sec} queries the
 
 [role="screenshot"]
-image::images/threat-intel-sub-sec.png[Enriched with Threat Intelligence subsection with matched file hash, 700]
+image::images/expanded-correlations-view.png[Expanded view of correlation data, 65%]
+
+In the expanded view, corelation data is organized into several tables:
 
-TIP: To learn more about the query, select the **Threat Intel** tab, then click the **Inspect** button next to the threat you want to inspect.
+* **Suppressed alerts**: Shows how many duplicate alerts were suppressed. This information only appears if alert suppression is enabled for the rule.
++
+preview::[]
+
+* **Related cases**: Shows cases to which the alert has been added. Click a case's name to open its details.
+* **Alerts related by source event**: Shows alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
+* **Alerts related by session**: Shows alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
+* **Alerts related by ancestry**: Shows alerts that are related by process events on the same linear branch. Note that alerts generated from processes on child or related branches are not shown. To further examine alerts, click *Investigate in timeline*.
 
-Additional threat intelligence data is generated for indicator match alerts when indicators are matched with source events, as outlined by the conditions in the relevant indicator match rule. Indicator match alerts always populate the *Threat Matched Detected* section under the *Threat Intel* tab with additional threat match data.
+[discrete]
+[[prevalence-overview]]
+=== Prevalence
 
-IMPORTANT: After upgrading to {stack} version 7.15.0 or newer from release versions 7.12.0 through 7.14.2, you need to migrate detection alerts enriched with threat intelligence data to ensure threat intelligence properly displays in {elastic-sec}. For more information, refer to instructions for <<post-upgrade-req-cti-alerts, migrating detection alerts enriched with threat intelligence data>>.
+The Prevalence overview shows whether data from the alert was frequently observed on other host events from the last 30 days. Prevalence calculations use values from the alert’s highlighted fields. Highlighted field values that are observed on less than 10% of hosts in your environment are considered uncommon (not prevalent) and are mentioned in the Prevalence overview. Highlighted field values that are observed on more than 10% of hosts in your environment are considered common (prevalent) and are not mentioned in the Prevalence overview. 
```

Comment:
```suggestion
The Prevalence overview shows whether data from the alert was frequently observed on other host events from the last 30 days. Prevalence calculations use values from the alert’s highlighted fields. Highlighted field values that are observed on less than 10% of hosts in your environment are considered uncommon (not prevalent), and appear in the Prevalence overview. 
```

Reply:
So, I want to keep both of these statements because users see information about host prevalence in both situations. To elaborate, if highlighted field values are observed frequently across multiple host events in the user's env, they should see this message:
<img width="460" alt="commonly-observed-host-values" src="https://github.com/elastic/security-docs/assets/79124755/059e38f0-96af-4cc6-8aac-dc31dd9d8a19">

(cc @PhilippeOberti and @christineweng to verify this. This is also another copy message we'll probably need to tweak in a future release because I wrote this based on [this Slack convo](https://elastic.slack.com/archives/C05QD3H5D9A/p1693421581093329), which includes host and user events in the logic/calculations for the Prevalence section)

If highlighted field values _aren't_ observed frequently across multiple host events in the user's env (i.e., the host events are uncommon), they should see something like this:

<img width="436" alt="uncommon-host-values" src="https://github.com/elastic/security-docs/assets/79124755/3272b3a0-bcbd-4d3f-8a15-1f8554b5cc96">

Reply:
Would it be more accurate or clear, if I modified the text as so:

```suggestion
The Prevalence overview shows whether data from the alert was frequently observed on other host events from the last 30 days. Prevalence calculations use values from the alert’s highlighted fields. Highlighted field values that are observed on less than 10% of hosts in your environment are considered uncommon (not prevalent) and are listed individually in the Prevalence overview. Highlighted field values that are observed on more than 10% of hosts in your environment are considered common (prevalent) and are described as frequently observed in the Prevalence overview. 
```

Reply:
Confirmed with @PhilippeOberti via Slack that the above text is accurate.